app/code/Magento/Paypal/Model/Payflow/Service/Response/Transaction.php

```diff
@@ -119,11 +119,13 @@ public function savePaymentInQuote($response, $cartId)
         $payment->setAdditionalInformation(Payflowpro::PNREF, $response->getData(Payflowpro::PNREF));
         $payment->setAdditionalInformation('result_code', $response->getData('result'));
 
-        $expDate = $response->getData('expdate');
-        $expMonth = $this->getCcExpMonth($expDate);
-        $payment->setCcExpMonth($expMonth);
-        $expYear = $this->getCcExpYear($expDate);
-        $payment->setCcExpYear($expYear);
+        if (is_numeric($response->getData('expdate')) {
```

Comment:
Why not store $response->getData('expdate') in a variable so it is not needed to call it twice?